gradle/libs.versions.toml

```diff
@@ -23,7 +23,7 @@ logback = "1.2.11"
 maven = "3.8.7"
 mavenResolver = "1.7.3"
 micrometer = "1.10.5"
-nessieClientVersion = "0.44.0"
+nessieClientVersion = "0.51.1" # Must be in sync with Nessie version in the Iceberg release.
```

Comment:
Could we pull this automatically from Iceberg dependencies?

Reply:
Should be possible. I just have to figure it out.

Reply:
yeah, maybe in a follow-up PR...